modules/email-chat/email-chat.service.js

```diff
@@ -13,22 +13,47 @@ const {
 const {
   DELIVERY_SERVICES_INITIAL_VALUES: { ID, DATE, INITIAL_TEXT, UKR, EN },
 } = require('../../consts/delivery-services');
+const { minDefaultDate } = require('../../consts/date-range');
 
 class EmailChatService {
-  async getAllEmailQuestions({ filter = {}, skip }) {
-    const { emailQuestionStatus } = filter;
+  async getAllEmailQuestions({ filter = {}, pagination: { skip, limit } }) {
+    const { emailQuestionStatus, search, date } = filter;
+    let maxDate = new Date();
+    let minDate = minDefaultDate;
 
-    const filters = emailQuestionStatus
+    const filterOptions = emailQuestionStatus
```

Comment:
please, rewrite this filter condition as you did bellow (in one style)

Reply:
Solved